e2e/explorer/workflow/diagramEditor.spec.ts

```diff
@@ -0,0 +1,295 @@
+import { Page, expect, test } from "@playwright/test";
+import { createNamespace, deleteNamespace } from "../../utils/namespace";
+
+import { consumeEvent as consumeEventWorkflow } from "~/pages/namespace/Explorer/Tree/NewWorkflow/templates";
+import { createRevision } from "~/api/tree/mutate/createRevision";
+import { createWorkflow } from "~/api/tree/mutate/createWorkflow";
+import { faker } from "@faker-js/faker";
+
+let namespace = "";
+let workflow = "";
+
+const getCommonPageElements = async (page: Page) => {
```

Comment:
All tests work on my system if I remove the async and awaits.

From what I've understood from the PW docs, these selectors are executed when you use them, so that's only later in the code and there is nothing to wait for at this point.